CMakeLists.txt

```diff
@@ -597,10 +597,15 @@ if (NOT MATH_LIBRARY)
     set(MATH_LIBRARY "")
 endif()
 
+if(APPLE)
+include(CheckSymbolExists)
+check_symbol_exists(lrint math.h FOUND_LRINT)
```

Comment:
Any reason to change from HAVE_LRINT to FOUND_LRINT? All of our other checks use "HAVE_" as far as I understand.

Reply:
It was the only I could make it work.